components/admin/requests/permit-holder-information/Form.tsx

```diff
@@ -57,6 +58,16 @@ export default function PermitHolderInformationForm(props: PermitHolderInformati
             />
           </FormControl>
 
+          {props.permitHolderInformation.type === 'NEW' && (
+            <FormControl>
+              <FormLabel>{'Middle name'}</FormLabel>
+              <Input
+                value={props.permitHolderInformation.middleName || ''}
+                onChange={handleChange('middleName')}
+              />
+            </FormControl>
+          )}
```

Comment:
Middle name field should always be present - I believe designs are a bit outdated